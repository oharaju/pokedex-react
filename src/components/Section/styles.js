import styled from 'styled-components';
import image from '../../images/wallpaper-florest.png';
import { device } from '../../utils/media'

export const Container = styled.div`
  background-image: url(${image});
  padding: 20px;
  height: 697px;
  font-size: 20px;
  border-radius: 10px;
  display: flex;
  flex-direction: column;
  align-items: center;
  overflow: auto;
  margin-left: auto;
  margin-right: auto;

  @media ${device.tablet} {
    width: 590px;
  }
`

export const Title =  styled.h1`
  font-family: 'Paytone One', sans-serif;
  font-size: 56px;
  font-weight: bold;
  color: #FFCC01;
  -webkit-text-stroke: 0.1rem #3264AF;
  margin-bottom: 40px;
`
