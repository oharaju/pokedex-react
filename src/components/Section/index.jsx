import { Container, Title } from './styles';
import Form from '../Form';

function Section() {
  return(
    <>
      <Container>
        <Title>Pok√©dex</Title>
        <Form />
      </Container>
    </>
  )
}

export default Section;