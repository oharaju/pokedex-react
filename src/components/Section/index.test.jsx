import renderer from 'react-test-renderer';
import Section from './index';

it('section structure', () => {
  const structure = renderer.create(<Section/>).toJSON();
  expect(structure).toMatchSnapshot();
});