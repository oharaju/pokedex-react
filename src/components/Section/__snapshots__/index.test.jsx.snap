// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`section structure 1`] = `
<div
  className="sc-beySbM iLMgTu"
>
  <h1
    className="sc-guDLRT ThJfX"
  >
    Pok√©dex
  </h1>
</div>
`;
